packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -320,7 +371,7 @@ export class SummaryManager extends EventEmitter implements IDisposable {
 
     private tryRestart(): void {
         const shouldSummarizeState = this.getShouldSummarizeState();
-        if (shouldSummarizeState.shouldSummarize && shouldSummarizeState.shouldStart) {
+        if (shouldSummarizeState.shouldSummarize) {
```

Comment:
>( [](http://example.com/codeflow?start=11&length=1)

this check is no longer complete, as it doesn't valid the client is actuall the client that should summarize. does that matter here?

Reply:
`shouldSummarize` is only true if it is the client that should summarize (i.e. it is elected). So this still holds the same as before. Previously `shouldStart` could be false if waiting for the previous lingering summarizer client to leave the quorum.